src/index.html

```diff
@@ -10,23 +10,45 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Airbnb Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__container">
+      <form
+        action="/"
+        method="get"
+        class="search-bar search-bar--big"
+        data-qa="big"
+      >
+        <div class="icon-big">
+          <input
+            type="text"
+            data-qa="keypress"
+            placeholder="Try “Los Angeles“"
+            class="search__input big"
```

Comment:
The attributes of the `input` element are not correctly formatted. Each attribute should start on a new line with a 2-space indentation.